src/app/api/urls/[id]/route.test.ts

```
@@ -0,0 +1,224 @@
+
+// Mock supabaseAdmin
+const mockSupabaseAdmin = {
+  from: jest.fn(() => ({
+    update: jest.fn().mockReturnThis(),
+    eq: jest.fn().mockReturnThis(),
+    select: jest.fn().mockReturnThis(),
+    single: jest.fn().mockReturnThis(),
+  }))
+};
+
+(supabaseAdmin as any) = mockSupabaseAdmin;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)